src/measure_theory/measure/hausdorff.lean

```diff
@@ -678,110 +763,48 @@ begin
       simp only [zero_le, measure_singleton] } }
 end
 
-end measure
-
-open_locale measure_theory
-open measure
-
-/-!
-### Hausdorff measure and Lebesgue measure
--/
+open_locale pointwise
 
-/-- In the space `ι → ℝ`, Hausdorff measure coincides exactly with Lebesgue measure. -/
-@[simp] theorem hausdorff_measure_pi_real {ι : Type*} [fintype ι] :
-  (μH[fintype.card ι] : measure (ι → ℝ)) = volume :=
+/-- This is a version of `lipschitz_on_with.hausdorff_measure_image_le` for dilations where the
```

Comment:
Only after having proved this the hard way do I realize I can just apply this lemma twice with antisymmetry...